apps/ledger-live-desktop/src/renderer/screens/platform/App.jsx

```diff
@@ -40,8 +40,8 @@ export default function PlatformApp({ match, appId: propsAppId, location }: Prop
 
   const returnTo = useMemo(() => {
     const params = new URLSearchParams(search);
-    return params.get("returnTo") || internalParams?.returnTo;
-  }, [search, internalParams]);
+    return urlParams?.returnTo || params.get("returnTo") || internalParams?.returnTo;
```

Comment:
We may be able to delete `params.get("returnTo") || internalParams?.returnTo` as I don't see any params being passed around LLD that way... but I haven't deleted them as I'm unsure if any deep links would have them.

Reply:
I would go for that instead, don't see why we would need the state here
```suggestion
    return internalParams?.returnTo || params.get("returnTo");
```

Reply:
We might want to still keep `params.get("returnTo")` but not sure about `internalParams`